Sources/CapriccioLib/SwiftTestsFilesWriter.swift

```diff
@@ -61,11 +65,21 @@ public final class SwiftTestsFilesWriter {
 
 public protocol CodeWriting {
     func write(code: String, toFile featureFilePath: String)
+    func writeAndCreate(code: String, toFile filePath: String, toDirectory directoryPath: String)
 }
 
 public final class CodeWriter: CodeWriting {
     public init() {}
     
+    public func writeAndCreate(code: String, toFile filePath: String, toDirectory directoryPath: String) {
```

Comment:
Is this function tested?

Reply:
Yes it is
<img width="993" alt="Screenshot 2019-07-12 at 13 36 01" src="https://user-images.githubusercontent.com/1630594/61128555-11b54a00-a4aa-11e9-9aa6-81c948511cef.png">